<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UARTDialog</class>
 <widget class="QDialog" name="UARTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UART connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupSpeed">
       <property name="title">
        <string>Speed (bps)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string notr="true">300</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="Index" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string notr="true">1200</string>
          </property>
          <property name="Index" stdset="0">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string notr="true">2400</string>
          </property>
          <property name="Index" stdset="0">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string notr="true">4800</string>
          </property>
          <property name="Index" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string notr="true">9600</string>
          </property>
          <property name="Index" stdset="0">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_6">
          <property name="text">
           <string notr="true">19200</string>
          </property>
          <property name="Index" stdset="0">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_7">
          <property name="text">
           <string notr="true">38400</string>
          </property>
          <property name="Index" stdset="0">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_8">
          <property name="text">
           <string notr="true">57600</string>
          </property>
          <property name="Index" stdset="0">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_19">
          <property name="text">
           <string notr="true">115200</string>
          </property>
          <property name="Index" stdset="0">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_20">
          <property name="text">
           <string notr="true">31250 (MIDI)</string>
          </property>
          <property name="Index" stdset="0">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupParity">
       <property name="title">
        <string>Data bits/parity</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_9">
          <property name="text">
           <string>8, None</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="Index" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_10">
          <property name="text">
           <string>8, Even</string>
          </property>
          <property name="Index" stdset="0">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_11">
          <property name="text">
           <string>8, Odd</string>
          </property>
          <property name="Index" stdset="0">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_12">
          <property name="text">
           <string>9, None</string>
          </property>
          <property name="Index" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupStop">
         <property name="title">
          <string>Stop bits</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButton_13">
            <property name="text">
             <string notr="true">1</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_14">
            <property name="text">
             <string notr="true">2</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPolarity">
         <property name="title">
          <string>Receive polarity</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="radioButton_18">
            <property name="text">
             <string>Idle 1</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_17">
            <property name="text">
             <string>Idle 0</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupOutput">
         <property name="title">
          <string>Output type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="radioButton_16">
            <property name="text">
             <string>Open drain (H=Hi-Z, L=GND)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="Index" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_15">
            <property name="text">
             <string>Normal (H=3.3V, L=GND)</string>
            </property>
            <property name="Index" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UARTDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UARTDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
